src/components/Admin/index.js

```diff
@@ -0,0 +1,89 @@
+import React, { Component } from 'react';
+import {
+  Container,
+  FormGroup,
+  Label,
+  Alert,
+  Button,
+  Jumbotron,
+  Row
+} from 'reactstrap';
+import bcrypt from 'bcryptjs';
+import '../../styles/ApproveIndex.css';
+import ApproveComponent from './ApproveComponent';
+
+class Approve extends Component {
+  constructor(props) {
+    super(props);
+    this.validatePassword = this.validatePassword.bind(this);
+    this.password = React.createRef();
+    this.state = {
+      isFirstLoad: true,
+      isAuthenticated: false
+    };
+  }
+
+  validatePassword() {
+    this.setState({ isFirstLoad: false });
+    var self = this;
+    bcrypt.compare(
+      this.password.current.value,
+      '$2a$10$O4SrNipdI3ChGKfxyTTpougsxZ85nTzyDcvhpCzL4hHAFV1UalHy2',
+      function(err, result) {
+        if (err) {
+          throw err;
+        }
+        if (result) {
+          self.setState({ isAuthenticated: true });
+        }
+      }
+    );
+  }
+
+  render() {
+    return (
+      <Container className="stories-page">
+        {!this.state.isAuthenticated && (
+          <Jumbotron className="float-center">
+            <FormGroup className="mb-2 mr-sm-2 mb-sm-0 ">
+              {!this.state.isFirstLoad && !this.state.isAuthenticated && (
+                <Alert color="danger">
+                  Incorrect Password! Please enter correct password to login
+                </Alert>
+              )}
+              <br />
+              {!this.state.isAuthenticated && (
+                <div>
+                  <h1 className="text-primary">
+                    Welcome to TruthTree Admin login!
+                  </h1>
+                  <Row style={{ paddingLeft: '25px' }}>
+                    <Label for="adminPassword" className="mr-sm-2">
+                      Please enter admin password for authentication
+                    </Label>
+                  </Row>
+                  <Row style={{ paddingLeft: '25px' }}>
+                    <input
```

Comment:
Can we use `reactstrap`'s `Input` component here?

Reply:
@jhalaa

Reply:
I was concerned because its styling looks off, it's not uniform with other input boxes we have. I think using `innerRef` would be the easiest, otherwise, if you decide to keep this, can you add bootstrap style into is `className` prop? I think just `form-control` would make it look nice again.